sdf/mandelbulbSDF.glsl

```diff
@@ -3,7 +3,11 @@
 /*
 original_author: Kathy McGuiness
 description: Returns the mandelbulb SDF
+For more information about the mandlebulb, check [this article](https://en.wikipedia.org/wiki/Mandelbulb)
```

Comment:
please respect the YAML structure of the comments. For multiple lines do:
```yaml
description: |
    Something blabla
    other line something bla bla
```

Reply:
I have updated the file.